prebid_cache_client/prebid_cache.go

```diff
@@ -112,3 +126,54 @@ func Put(ctx context.Context, objs []*CacheObject) error {
 
 	return nil
 }
+
+func PutXml(ctx context.Context, objs []*CacheXmlObject) error {
+	if len(objs) == 0 {
```

Comment:
is this check needed? looks like you already checked that objects being out are > 0

Reply:
If some other people is trying to use this func, maybe it's safe to add this check

Reply:
Also, looks like this check is what @dbemiller added to the original Put method for a bug fix...